Doc/c-api/code.rst

```diff
@@ -1,9 +1,9 @@
 .. highlight:: c
 
-.. _codeobjects:
-
 .. index:: object; code, code object
 
+.. _codeobjects:
+
```

Comment:
How does this change affect `:ref:` and index entries linking (if it does)?

Reply:
It doesn't affect the index entries (it still links to the same place, right above the section) but fixes an issue with the ref target not actually being the section as intended, and thus not having a generated link text causing ``:ref:`codeobjects` `` to error out. It won't affect any existing refs, since it only changes the generated link text and not the ref target location, which the existing refs could not be using or they would have errored and failed the build.